Incorrect user is shown when accessing a particular user via system admin tab

Go to Admin -> organizations, then select an organization and select a user

It will open a new page where it tells you the information about the user, but the user it shows you is not the correct user - if you look in the URL you can see the id that the system looked up.

The system looked up the value stored under id in the table 'user_organizations' rather than looking up the value of user_id in that same table as it should have done. Therefore, an incorrect user, or in some cases no user, is shown.

When creating a user via the (system) "Admin" page, upon saving it gives
`error: "Name not set."`

In console it gives
`jquery-3.5.1.js:10099 POST http://localhost/rest/users 400 (Bad Request)`
`restClient.js:44 Error occured in REST client, when sending POST to url: /rest/users`
`user data not posted`

Nothing changes in the databases.

--- a/ui/src/js/viewModels/systemAdminOrganizations.js
+++ b/ui/src/js/viewModels/systemAdminOrganizations.js
@@ -127,8 +127,17 @@ define(['appController','utils','ojs/ojrouter','ojs/ojcore', 'knockout', 'jquery
 
                         self.handleUserRowChanged = function (event) {
                             if (event.detail.value[0] !== undefined) {
-								router.go('user/' + event.detail.value[0].startKey.row);
-
+                                //GET /rest/user_organizations/{user organization id} - REST
+                                return $.when(restClient.doGetJson('/rest/user_organizations/' + event.detail.value[0].startKey.row)
+                                .then(
+                                    success = function (response) {
+                                        router.go('user/' + response.user_id);
+                                    },
+                                    error = function (response) {
+                                        console.log("User organizations not loaded");
+                                        self.userOrgValid(false);
+                                    })
+                                );
                             }
                         };
                     }();
@@ -138,7 +147,7 @@ define(['appController','utils','ojs/ojrouter','ojs/ojcore', 'knockout', 'jquery
 						return $.when(restClient.doGetJson('/rest/organizations/' + orgId + '/user_organizations')
 							.then(
 								success = function (response) {
-									self.userOrgValues(response.user_organizations);
+                                    self.userOrgValues(response.user_organizations);
 									var userOrgs = self.userOrgValues().filter(function(user){
 										user.adminTable = [];
 										if (user.admin === "Y") {

